implementations/CMIP6-UofT/add_CMIP6.py

```diff
@@ -90,7 +91,8 @@ class STACItem(BaseModel):
 def make_cmip6_id(attrs: MutableMapping[str, Any]) -> str:
     """Return unique ID for CMIP6 data collection (multiple variables)."""
     keys = ["activity_id", "institution_id", "source_id", "experiment_id", "variant_label", "table_id", "grid_label",]
-    return "_".join(attrs[k] for k in keys)
+    item_name = "_".join(attrs[k] for k in keys)
+    return hashlib.md5(item_name.encode("utf-8")).hexdigest()
```

Comment:
I've made a change to this function in https://github.com/crim-ca/stac-populator/pull/16. I included variable_id in the list so that the entire string becomes unique. I believe that string is what ESGF uses to uniquely identify each dataset. My thinking was that we could use the same string for the ID here. Is there an advantage to using a hash rather than a human readable string?

Reply:
One question I had was whether we wanted some form of variable aggregation at the item level. That is, I suspect one item could have different variables listed as assets, instead of having one item per variable. Not sure in practice what difference it would make, but I think this is worth testing from the client side. 

Unclear for me as well. I initially just copied the approach used for the collections. Playing with the clients, I realized that hashes for collections are really not practical. For items, I'm not sure they need to be human readable. I suspect the selection would be based on a search of properties, not parsing the item ID. So no strong opinion on this yet.